lightning/src/util/persist.rs

```diff
@@ -406,30 +406,62 @@ where
 		)?;
 		let mut res = Vec::with_capacity(monitor_list.len());
 		// for each monitor...
```

Comment:
Comments in this function don't seem very useful anymore.

Reply:
yeah meant to go back and clean those up thx